astroquery/hitran/reader.py

```diff
@@ -123,6 +261,43 @@ def quanta_formatter(group_global='class1', group_local='group1'):
 
     return loc,glob
 
+def download_hitran(m, i, numin, numax):
+    """
+    Download HITRAN data for a particular molecule. Based on fetch function from
+    hapi.py.
+
+    Parameters
+    ----------
+    m : int
+        HITRAN molecule number
+    i : int
+        HITRAN isotopologue number
+    numin : real
+        lower wavenumber bound
+    numax : real
+        upper wavenumber bound
+    """
+    iso_id = str(ISO[(m,i)][ISO_INDEX['id']])
+    mol_name = ISO[(m,i)][ISO_INDEX['mol_name']]
+    url = HITRAN_URL + '/lbl/api?' + \
+        'iso_ids_list=' + iso_id + '&' + \
+        'numin=' + str(numin) + '&' + \
+        'numax=' + str(numax)
+    try:
+        req = urllib2.urlopen(url)
```

Comment:
That is a good suggestion, will change it. This should also be python 2 and 3 compatible but have not tested it.